Mathlib/Algebra/Order/GroupWithZero/Canonical.lean

```diff
@@ -32,7 +32,7 @@ variable {α : Type*}
 
 /-- A linearly ordered commutative monoid with a zero element. -/
 class LinearOrderedCommMonoidWithZero (α : Type*) extends LinearOrderedCommMonoid α,
-  CommMonoidWithZero α where
+  CommMonoidWithZero α, OrderBot α where
   /-- `0 ≤ 1` in any linearly ordered commutative monoid. -/
   zero_le_one : (0 : α) ≤ 1
```

Comment:
Maybe not in scope, but could this also extend `ZeroLEOneClass`?

Reply:
Looks complicated because it makes `LinearOrderedCommMonoidWithZero.mk` require a `[Zero α]` argument that doesn't play well with `where` and `{ }` notations